aws/extensions/computetask_awswin_cwlog/extension.ftl

```diff
@@ -33,6 +33,16 @@
     [#return retval ]
 [/#function]
 
+[#function nix_path_converter win_path ]
+    [#local seqWin = win_path?split(r"\")]
+    [#if (seqWin[0])?matches('[c-zC-Z]:')]
+        [#local seqWin = [(seqWin[0])?keep_before(":")] + seqWin[1..]]
```

Comment:
Why not 

path?replace(":", "") 

That would remove any reference to : which is not allowed at anypoint in the logstream name?

Reply:
Fixed and moved to using native object support to handle escaping and make it easier to follow the extension 

see 4a9fa5f